invenio_pidstore/models.py

```diff
@@ -55,12 +55,12 @@
 from datetime import datetime
 
 from sqlalchemy.exc import SQLAlchemyError
+from invenio_db import db, InvenioDB
 
-from invenio_base.globals import cfg
-from invenio_ext.sqlalchemy import db
 from invenio_utils.text import to_unicode
```

Comment:
``six.text_type(foo)`` seems to work. It does not support accents, but I think I can assume ``pid_value`` and ``object_value`` won't contain any.